src/store/store.ts

```diff
@@ -27,38 +27,50 @@ interface State {
 interface Actions {
   updateCurrentProject: (project: CurrentProjectResponse) => void
   navigateTo: (nextPage: State["currentPage"]) => void
-  updateUserSession: (newLoggedStatus: boolean) => void
+  updateUser: (newUser: UserSession) => void
+  resetStore: () => void
+}
+
+const initialState: State = {
+  user: {
+    id: undefined,
+    username: undefined,
+    email: undefined,
+    audience_member: {
+      is_quiet: false,
+      is_paused: false,
+      project_id: undefined,
+      current_index: 0,
+      event_time: "12:00:00.000",
+      playlist: []
+    }
+  },
+  active_project: undefined,
```

Comment:
so it turns out you are using this to keep the entire current project object ?

I removed it and it broke the homepage because you use it when you fetch the current project the first time